curriculum/challenges/english/01-responsive-web-design/basic-css/import-a-google-font.english.md

```diff
@@ -21,7 +21,7 @@ Family names are case-sensitive and need to be wrapped in quotes if there is a s
 
 ## Instructions
 <section id='instructions'>
-Create a <code>font-family</code> CSS rule that uses the <code>Lobster</code> font, and ensure that it will be applied to your <code>h2</code> element.
+Import the Lobster font to your web page. Then use an <code>h2</code> element selector to create a <code>font-family</code> CSS rule that uses the Lobster font to ensure it will be applied to your <code>h2</code> element.
```

Comment:
```suggestion
Import the Lobster font to your web page. Then, use an element selector to set <code>Lobster</code> as the <code>font-family</code> for your <code>h2</code> element.
```

Reply:
I'm fine with this if you are @RandellDawson

Reply:
@moT01 Go ahead and commit that suggestion or create a new commit.  @Sky020 and I will be the reviewers/approvers for this PR since you are adding commits.